.gitignore

```diff
@@ -0,0 +1 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
My PhpStorm project root is never the Git repository itself.

Also helpful: http://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file